package.build.json

```diff
@@ -10,24 +7,5 @@
     "jest-summary-reporter": "^0.0.2",
     "jest-environment-node": "^26.6.2",
     "cross-env": "^7.0.2"
-  },
-  "workspaces": {
-    "packages": [
-      "client",
-      "core",
-      "fullstack",
-      "interfaces",
-      "plugin-utils",
-      "plugins/*",
-      "puppet",
-      "puppet-chrome",
-      "mitm",
-      "mitm-socket",
-      "testing",
-      "../../commons/build"
-    ]
-  },
```

Comment:
Shouldn't workspaces be left in the package.build.json? How else will it find the packages when `yarn build` locally?

Reply:
I changed prepare-build and prepare-dist to automatically resolve the nested packages from the regular package.json

Reply:
Sorry, backing up, this pr changes the root package.json, package.build.json, package.dist.json to behave like the nested projects - the .build.json and .dist.json files just provide the overrides

Reply:
got it.